### ISSUE TITLE
No keyword argument neutral_losses for annotate_peptide_fragments

### ISSUE THREAD

**Issue by Trulsson** [bittremieux/spectrum_utils#21]
print(inspect.signature(spectrum.annotate_peptide_fragments))

Returns

(fragment_tol_mass: float, fragment_tol_mode: str, ion_types: str = 'by', max_ion_charge: Union[int, NoneType] = None, peak_assignment: str = 'most_intense') -> 'MsmsSpectrum'

Was this feature removed?

**Comment by bittremieux** [comment]
No, the opposite actually. Annotating neutral loss peaks is not included in the latest release on PyPI (v0.3.5) yet. It will be included in spectrum_utils version 0.4.0, but I want to add [ProForma](https://www.psidev.info/proforma) support as well before releasing a new version.

If you want to annotate neutral loss peaks, for the moment you can install the development spectrum_utils version directly from GitHub:
```
pip install git+https://github.com/bittremieux/spectrum_utils.git
```

Apologies for the inconvenience.

**Comment by Trulsson** [comment]
I see! Thanks for the quick reply, most appreciated.
